README-idrange.md

```diff
@@ -0,0 +1,171 @@
+Idrange module
+============
+
+Description
+-----------
+
+The idrange module allows the management of ID ranges.
```

Comment:
It might be good to add information about the use case of the module. This should be done for other modules also.

Reply:
Added use cases available in IPA command line through `ipa help idrange`.